service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/ActionCaseManagementOrderController.java

```diff
@@ -52,14 +46,12 @@
 public class ActionCaseManagementOrderController {
     private final DraftCMOService draftCMOService;
```

Comment:
Still not liking how this is also reliant on the draft service, thoughts on moving those that are needed for both draft and action to cmo service?